docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -0,0 +1,53 @@
+.. include:: ../../../03-exports/aliases.include
+.. include:: ../../../03-exports/aliases-api.include
+.. include:: ../../../03-exports/roles.include
+
+.. _statistics_domainparticipant:
+
+Statistics Domain Participant
+=============================
+
+In order to start collecting data in one of the previous topics (:ref:`statistics_topic_names`), the corresponding
+statistics DataWriter should be enabled.
+In fact, *Fast DDS Statistics module* can be enabled and disabled at runtime.
+For this purpose, *Fast DDS Statistics module* exposes an extended DDS |StatisticsDomainParticipant-api| API:
+
+Enable statistics DataWriters
+-----------------------------
+
+.. warning::
+    This method will be implemented in future releases. For the moment it returns always ``RETCODE_UNSUPPORTED``.
+
+Statistic DataWriters are enabled using the method |enable_statistics_datawriter|.
+This method requires as parameters:
+
+* Name of the statistics topic to be enabled (see :ref:`statistics_topic_names` for the statistics topic list).
+* DataWriter QoS profile (see :ref:`statistics_datawriter_qos` for the recommended profile).
+
+Disable statistics DataWriters
+------------------------------
+
+.. warning::
+    This method will be implemented in future releases. For the moment it returns always ``RETCODE_UNSUPPORTED``.
+
+Statistic DataWriters are disabled using the method |disable_statistics_datawriter|.
+This method requires as parameter:
+
+* Name of the statistics topic to be disabled (see :ref:`statistics_topic_names` for the statistics topic list).
+
+.. warning::
+    Before deleting the |StatisticsDomainParticipant-api|, remember to ensure that all statistics DataWriters have been
+    disabled. Otherwise, |DomainParticipantFactory::delete_participant-api| will return
+    ``RETCODE_PRECONDITION_NOT_MET``.
+
```

Comment:
We should probably remove this warning. If the user can state several statistics DataWriters to be enabled by default when the domainParticipant is created and enabled, the user should not be concerned with manually removing these statistics DataWriters before deleting the domainParticipant. The statistics DataWriters should probably be removed directly by Fast DDS in case they have been created.

Reply:
We have finally decided that deleting the participant will automatically delete the statistics DataWriters